Cargo.toml

```diff
@@ -20,6 +20,7 @@ members = [
   "src/base",
   "src/gax",
   "src/generated/api",
+  "src/generated/bigtableadmin/v2",
```

Comment:
this is surprising to me. I expected `src/generated/bigtable/admin/v2`

Reply:
Is it based on the endpoint?

If it is based on the `*.yaml`... I think there are services with multiple yamls of the same name in different directories.

Reply:
> services with multiple yamls of the same name in different directories

It is `monitoring`, which I think is next on your list anyway.

```
google/monitoring/dashboard/v1/monitoring.yaml
google/monitoring/metricsscope/v1/monitoring.yaml
google/monitoring/v3/monitoring.yaml
```

Reply:
The directory can change without much problem.  Customers never see it.  I am agnostic about its name so I just changed it.

The interesting part is the package name.  It is  `${prefix}-bigtable-admin-v2`, which is based on the protobuf package name.

Reply:
> The directory can change without much problem. Customers never see it.

Ah, ok.